app/models/application_record.rb

```diff
@@ -1,3 +1,42 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+
+  def set_bad_method
+    not_used = true
+    @not_great = {
+      :at_ruby_2 => true,
+      :at_strings => "Double quotes are bad!",
```

Comment:
Bliss.AI: report

| Debt | Issue |
|---:|---|
| 0.5 | Use the new Ruby 1.9 hash syntax. |
| 0.5 | Prefer single-quoted strings when you don't need string interpolation or special symbols. |